acpreprocessing/stitching_modules/convert_to_n5/tiff_to_ngff.py

```diff
@@ -639,11 +905,14 @@ class N5GenerationParameters(argschema.schemas.DefaultSchema):
     chunk_size = argschema.fields.Tuple((
         argschema.fields.Int(),
         argschema.fields.Int(),
-        argschema.fields.Int()), required=False, default=(64, 64, 64))
+        argschema.fields.Int(),
```

Comment:
Does this break functionality with 3 dimensional arrays?  Should we make `DownsampleOptions3D` and `DownsampleOptions5D` subclasses?

Reply:
I'm not sure, but I suppose it could? I assumed the input to the tiff to ngff process would be the same as tiff stacks read into 3D arrays, but since the omezarr format expects 5D chunks accounting for time and channel in addition to spatial indices, I specifically added dimensions for the omezarr conversion.
Alternatively would it make sense to converge the n5 format to also include the time and channel dimensions in the chunking?

Reply:
I am not sure if adding dimensions to n5 works for e.g. the existing stitching-spark code.  I don't think we should break the 3D writing just yet, even if it means making more specific schemas that we end up making the default.

Reply:
How does this look to you currently? It feels unnatural to separate chunk_size from the NGFF parameters, but isn't this the only substantive difference in the schemas between N5 and Zarr?